tests/test_tifs.py

```diff
@@ -34,7 +34,7 @@ def _get_tif_tolerances(file_name: str):
     if file_name.endswith('vert_disp.tif'):
         rtol, atol = 0.0, 1.1
     if file_name.endswith('los_disp.tif'):
-        rtol, atol = 0.0, 1e-01
+        rtol, atol = 0.0, 1.5e-01
```

Comment:
I assume this is because we need to be  little looser here for InSAR than RTC?

Reply:
Copernicus DEM for InSAR introduced slightly more variance in the los_disp.tif for one of our specific test cases.  This should have been adjusted immediately after that release, but I let it languish.